src/test/java/bio/terra/service/dataset/flight/ingest/IngestUtilsTest.java

```diff
@@ -111,17 +112,17 @@ public void testNoDoubleDash() {
 
   @Test(expected = InvalidIngestStrategyException.class)
   public void testMaxLineIngestCheck() {
-    int numLines = 11;
-    when(applicationConfiguration.getMaxDatasetIngest()).thenReturn(10);
+    long numLines = 11L;
+    when(configurationService.getParameterValue(ConfigEnum.LOAD_BULK_FILES_MAX)).thenReturn(10L);
     IngestUtils.checkForLargeIngestRequests(
-        numLines, applicationConfiguration.getMaxDatasetIngest());
+        numLines, configurationService.getParameterValue(ConfigEnum.LOAD_BULK_FILES_MAX));
   }
 
   @Test
   public void testSmallLineIngestCheck() {
-    int numLines = 9;
-    when(applicationConfiguration.getMaxDatasetIngest()).thenReturn(10);
+    long numLines = 9L;
+    when(configurationService.getParameterValue(ConfigEnum.LOAD_BULK_FILES_MAX)).thenReturn(10L);
```

Comment:
`ConfigEnum.LOAD_BULK_FILES_MAX` is removed in this PR so I don't know if mocking this makes sense anymore

Reply:
Looks like `checkForLargeIngestRequests` is only used in these tests now too so we could take that out

Reply:
Oh very true!